pkg/aws_keyhub/keyhub.go

```diff
@@ -134,20 +134,22 @@ func ExchangeToken(loginResponse LoginResponse) ExchangeResponse {
 	config := getAwsKeyHubConfig()
 	httpClient := getHTTPClient()
 
-	exchangePath := "/login/oauth2/exchange"
+	exchangePath := "/login/oauth2/token"
 	data := url.Values{
 		"grant_type":         {"urn:ietf:params:oauth:grant-type:token-exchange"},
 		"subject_token_type": {"urn:ietf:params:oauth:token-type:access_token"},
 		"subject_token":      {loginResponse.AccessToken},
+		"requested_token_type":{"urn:ietf:params:oauth:token-type:saml2"},
 		"resource":           {config.Keyhub.AwsSamlClientId},
+		"client_id":          {config.Keyhub.ClientId},
 	}
-	logrus.Debugln("KeyHub token exchange POST formdata:", data)
```

Comment:
De debug statement hier is wel weggevallen, lijkt me handig om te bewaren

Reply:
fixed